plugins/VstBase/VstPlugin.cpp

```diff
@@ -586,7 +580,7 @@ void VstPlugin::savePreset()
 	presName.replace("\"", "'"); // QFileDialog unable to handle double quotes properly
 
 	gui::FileDialog sfd( nullptr, tr( "Save Preset" ), presName.section(": ", 1, 1) + tr(".fxp"),
-		tr( "Vst Plugin Preset (*.fxp *.fxb)" ) );
+		tr( "VST Plugin Preset (*.fxp *.fxb)" ) );
```

Comment:
You fix spacing on everything else but this one? 😂

Reply:
The `savePreset()` function was longer than `openPreset()`, so I did not want to touch it as I was afraid it would lead to a style-update chain reaction... :smile:

Reply:
There, the function should be fixed now. :)

Reply:
oh hah, you did the entire function. Just meant the line itself like the others. ;P

Reply:
> like the others

In the first commit I also updated the surrounding code in `openPreset()`, and a few lines around the "Mutliple" typo. I didn't want to leave it as a wild mix of two different styles, so I tried to at least update everything in related blocks of code. Sometimes it's just hard to decide where to stop. :))

Reply:
Usually just the line touched is what we go for on updating style. 👌 All good though!

Reply:
OK, I'll limit the changes next time. Probably makes sense; the diff would be easier to review as it would contain only relevant changes.